### ISSUE TITLE
Implement additional functionalities of states

### ISSUE THREAD

**Issue by isaikatofficial** [Saikat-98/World-Covid-Tracker#3]
Additional feature of states need to be implemented

**Comment by smriti1313** [comment]
Hi, I would like to work on this. Is there any chance for it? Kindly let me know. thank you.

**Comment by isaikatofficial** [comment]
Sure, go ahead

**Comment by smriti1313** [comment]
Can you elaborate what needs to be done in this issue so that I can get a clear idea!?

**Comment by isaikatofficial** [comment]
Yes, I am looking for the cases for each region or states in each country, maybe like WB, Karnataka for India or any other state, like some Api which would provide me such stuff
